templates/support/livepatch.html

```diff
@@ -0,0 +1,160 @@
+    <div class="col-10">
+      <h2>Subscribe to Livepatch</h2>
+      <p class="u-no-max-width">
+        <a class="p-button--positive p-button--wide"  href="https://auth.livepatch.canonical.com/"><span class="p-link--external">Free for up to 3 machines</span></a>
+        <span style="padding-left: 2rem;">All you need is an Ubuntu One account.</span>
+      </p>
+      <p class="u-no-max-width">
+        <a class="p-button--positive p-button--wide"  href="https://aws.amazon.com/marketplace/seller-profile?id=565feec9-3d43-413e-9760-c651546613f2"><span class="p-link--external">AWS Marketplace</span></a> <span style="padding-left: 2rem;">Part of Ubuntu Advantage for AWS.</span></p>
+      <p class="u-no-max-width">
+        <a class="p-button--positive p-button--wide"  href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/Canonical.UbuntuAdvantageSupport?src=blogazure&tab=Overview"><span class="p-link--external">Azure Marketplace</span></a>
+        <span style="padding-left: 2rem;">Part of Ubuntu Advantage for Azure.</span>
+      </p>
+      <p class="u-no-max-width">
+        <a class="p-button--positive p-button--wide"  href="https://buy.ubuntu.com"><span class="p-link--external">Buy direct</span></a>
+        <span style="padding-left: 2rem;">Part of Ubuntu Advantage for desktops or datacenters.</span>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <h2 class="p-muted-heading u-align-text--center">Used by</h2>
+    <div class="col-12 prefix-1">
+      <ul class="row">
+        <li class="col-2 mobile-col-2 tablet-col-2"><img src="{{ ASSET_SERVER_URL }}9ae8ff55-2018-logo-bloomberg.svg" alt="Bloomberg" /></li>
+        <li class="col-2 mobile-col-2 tablet-col-2"><img src="{{ ASSET_SERVER_URL }}d8f890fb-logo-at%26t.svg" alt="AT&T" /></li>
+        <li class="col-2 mobile-col-2 tablet-col-2"><img src="{{ ASSET_SERVER_URL }}ff209af1-2018-logo-walmart.svg" alt="Walmart" /></li>
+        <li class="col-2 mobile-col-2 tablet-col-2"><img src="{{ ASSET_SERVER_URL }}d199354d-2018-logo-deutsche-telekom.svg" alt="Deutcshe Telecom" /></li>
+        <li class="col-2 mobile-col-2 tablet-col-2"><img src="{{ ASSET_SERVER_URL }}675af314-2018-logo-eBay.svg" alt="eBay" /></li>
+      </ul>
+      <ul class="row u-align--center">
+        <li class="col-2 mobile-col-2 tablet-col-2"><img src="{{ ASSET_SERVER_URL }}8a4d26de-2018-logo-cisco.svg" alt="Cisco" /></li>
+        <li class="col-2 mobile-col-2 tablet-col-2"><img src="{{ ASSET_SERVER_URL }}94da9f2a-2018-logo-NTT.svg" alt="NTT" /></li>
+        <li class="col-2 mobile-col-2 tablet-col-2"><img src="{{ ASSET_SERVER_URL }}dcde7628-2018-logo-best-buy.svg" alt="Best Buy" /></li>
+        <li class="col-2 mobile-col-2 tablet-col-2"><img src="{{ ASSET_SERVER_URL }}9f7bc2d4-2018-logo-paypal.svg" alt="PayPal" /></li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep">
+  <div class="row u-vertically-center">
```

Comment:
Vertically centre should be on the centred container. This row should have `u-equal-height`

Reply:
done